scripts/detectNotUsingGitPush1by1.fsx

```diff
@@ -563,37 +563,441 @@ let prCommits =
         .Split "\n"
     |> Seq.tail
 
-let notUsingGitPush1by1 =
-    prCommits
-    |> Seq.map(fun commit ->
-        use client = new HttpClient()
-        client.DefaultRequestHeaders.Accept.Clear()
-
-        client.DefaultRequestHeaders.Accept.Add(
-            MediaTypeWithQualityHeaderValue "application/vnd.github+json"
-        )
+type checkSuitsType =
```

Comment:
@realmarv typo ^ also: types should be PascalCase

Reply:
Done

Reply:
this typo is not addressed, read the message again: I'm pointing out 2 things, not 1